src/components/UserLoginForm/UserLoginForm.js

```diff
@@ -31,8 +33,16 @@ const UserLoginForm = ({ submitLogin }) => {
                 value={userCredentials}
                 onChange={(event) => setUserCredentials(event.target.value)}
             />
+            <input
+                className='password-input'
+                type="text"
+                name="userPassword"
+                placeholder="Enter your password"
+                value={userPassword}
+                onChange={() => setUserPassword("welcome2022")}
+            />
 
-            <button className='login-button' type='submit' disabled={!userCredentials}>Login</button>
+            <button className='login-button' type='submit' disabled={!userCredentials && !userPassword}>Login</button>
```

Comment:
this is requiring a password to sign in...  you said in slack that this isnt working yet so I dont know if we should do this especially for testing purposes.